src/client/Client.js

```diff
@@ -337,6 +343,83 @@ export default class Client {
         return this;
     }
 
+    /**
+     * @param {?number} minBackoff
+     * @returns {this}
+     */
+    setMinBackoff(minBackoff) {
+        if (minBackoff == null) {
+            throw new Error("minBackoff cannot be null.");
+        }
+        if (minBackoff > this._maxBackoff) {
+            throw new Error("minBackoff cannot be larger than maxBackoff.");
+        }
+        this._minBackoff = minBackoff;
+        return this;
+    }
+
+    /**
+     * @returns {number}
+     */
+    get minBackoff() {
+        return this._minBackoff;
+    }
+
+    /**
+     * @param {?number} maxBackoff
+     * @returns {this}
+     */
+    setMaxBackoff(maxBackoff) {
+        if (maxBackoff == null) {
+            throw new Error("maxBackoff cannot be null.");
+        } else if (maxBackoff < this._minBackoff) {
+            throw new Error("maxBackoff cannot be smaller than minBackoff.");
+        }
+        this._maxBackoff = maxBackoff;
+        return this;
+    }
+
+    /**
+     * @returns {number}
+     */
+    get maxBackoff() {
+        return this._maxBackoff;
+    }
+
+    /**
+     * @private
+     * @param {?number} minBackoff
+     * @param {?number} maxBackoff
+     * @returns {this}
+     */
+    // @ts-ignore
```

Comment:
We shouldn't be using `ts-ignore`. 
Also, unfortunately adding `@private` is not all you have to do to make something look private. You'll need to add `_` prefix to the method name. Something like `_setBackoff(...)`. If someone uses this SDK in JS they won't realize `setBackoff()` is private, but adding an `_` in the beginning is a recognized convention for private methods/fields.